ibazel/ibazel.go

```diff
@@ -37,9 +42,11 @@ var osExit = os.Exit
 var bazelNew = bazel.New
 var commandDefaultCommand = command.DefaultCommand
 var commandNotifyCommand = command.NotifyCommand
+var mrunToFiles = flag.Bool("mrunToFiles", false, "Log mrun to file for simpler log reading")
```

Comment:
the need for this feature makes me wonder, if you're going to open two or more log files to see the output, why not just run different `ibazel` commands in one or more terminals?

I think the answer is that it's hard to script around "start these three programs in three terminals" in any portable way...

Reply:
I think we are providing this option for those who want to see the outputs, while others who don't need the outputs can use just one terminal for `ibazel`. The other reason is that if you're running ten targets but only need two log files, you can just have two terminals instead of ten.